spec/lib/manageiq/api/common/user_spec.rb

```diff
@@ -0,0 +1,81 @@
+# Encoded Identity header user keys/values
+#{
+#  "identity": {
+#    "account_number": "0369233",
+#    "type": "User",
+#    "user" : {
+#      "username": "jdoe",
+#      "email": "jdoe@acme.com",
+#      "first_name": "John",
+#      "last_name": "Doe",
+#      "is_active": true,
+#      "is_org_admin": false,
+#      "is_internal": false,
+#      "locale": "en_US"
+#    },
+#    "internal" : {
+#      "org_id": "3340851",
+#      "auth_type": "basic-auth",
+#      "auth_time": 6300
+#     }
+#  }
+describe ManageIQ::API::Common::User do
+  let(:encoded) { {"x-rh-identity"=>"eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjAzNjkyMzMiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsidXNlcm5hbWUiOiJqZG9lIiwiZW1haWwiOiJqZG9lQGFjbWUuY29tIiwiZmlyc3RfbmFtZSI6IkpvaG4iLCJsYXN0X25hbWUiOiJEb2UiLCJpc19hY3RpdmUiOnRydWUsImlzX29yZ19hZG1pbiI6ZmFsc2UsImlzX2ludGVybmFsIjpmYWxzZSwibG9jYWxlIjoiZW5fVVMifSwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMzM0MDg1MSIsImF1dGhfdHlwZSI6ImJhc2ljLWF1dGgiLCJhdXRoX3RpbWUiOjYzMDB9fX0="} }
```

Comment:
@syncrou Should we use a Factory for this and create normal_user and admin user traits?